### ISSUE TITLE
Remove `/moveto` command

### ISSUE THREAD

**Issue by joesanford** [pyslackers/sirbot-pythondev#92]
While originally well-intentioned, the `/moveto #channel` command seems to tend to stifle conversation instead of moving it to a better channel. IMO it should be removed, and people should just ask kindly for the conversation to move. As @mrasband put it, it's a bit cold.

**Comment by mattrasband** [comment]
Agreed on this, it tends to totally shut down the conversation rather than move it. If nothing else, we should consider alternatives. I think right now maybe `/moveto` could auto-generate a message from the current user and post as them instead (or just request people to nicely ask)? Something like:

```
Sorry to ask this, but it seems this conversation has gone off topic and is conflicting with ongoing on-topic conversations here. Can you please move to #<channel>
```
